OpenRA.Game/Settings.cs

```diff
@@ -284,6 +284,11 @@ public class KeySettings
 		public Hotkey ReplaySpeedFastKey = new Hotkey(Keycode.F7, Modifiers.None);
 		public Hotkey ReplaySpeedMaxKey = new Hotkey(Keycode.F8, Modifiers.None);
 
+		public Hotkey NextTrack = new Hotkey(Keycode.AUDIONEXT, Modifiers.None);
```

Comment:
Also, I changed the default keys to numpad keys, because my keyboard doesn't have multimedia keys.